runtime/vm/CRIUHelpers.cpp

```diff
@@ -476,4 +495,44 @@ runDelayedLockRelatedOperations(J9VMThread *currentThread)
 	return rc;
 }
 
+BOOLEAN
+delayedLockingOperation(J9VMThread *currentThread, j9object_t instance, UDATA operation)
```

Comment:
we can leave this where it is and add a c wrapper for it. Example is in ValueTypeHelpers.cpp

Reply:
Replaced it w/ `VM_CRIUHelpers::delayedLockingOperation()`.